packages/react/src/Trans.tsx

```diff
@@ -35,8 +36,23 @@ export type TransProps = {
 } & TransRenderCallbackOrComponent
 
 export function Trans(props: TransProps): React.ReactElement<any, any> | null {
-  const { i18n, defaultComponent } = useLingui()
-  const { render, component, id, message, formats } = props
+  const lingui = useLingui()
+  return React.createElement(TransNoContext, { ...props, lingui })
```

Comment:
there is a bit of mess with `component` and `defaultComponent`. It would look weird when they would be on the same level in props. I just wanted to save structure as it was in context.